tower_cli/models/base.py

```diff
@@ -1189,6 +1189,30 @@ def cancel(self, pk=None, fail_if_not_running=False, **kwargs):
         # Return a success.
         return {'status': 'canceled', 'changed': changed}
 
+    @resources.command
+    def relaunch(self, pk=None, **kwargs):
+        """Relaunch a stopped job.
+
+        Fails with a non-zero exit status if the job cannot be relaunched.
+        You must provide either a pk or parameters in the job's identity.
+        """
+        # Search for the record if pk not given
+        if not pk:
+            existing_data = self.get(**kwargs)
+            pk = existing_data['id']
+
+        relaunch_endpoint = '%s%s/relaunch/' % (self.endpoint, pk)
+        data = {'extra_vars': {}}
+        # Attempt to relaunch the job.
+        try:
+            client.post(relaunch_endpoint, data=data)
+            changed = True
+        except exc.MethodNotAllowed:
+            changed = False
+
+        # Return a success.
+        return {'status': 'relaunched', 'changed': changed}
```

Comment:
If you do the ^ stuff, then best to remove "status" from this, so we use the API's value.